src/Subscription.php

```diff
@@ -791,7 +789,6 @@ protected function mergeItemsThatShouldBeDeletedDuringSwap(Collection $items)
     protected function getSwapOptions(Collection $items, array $options = [])
     {
         $payload = [
-            'automatic_tax' => $this->automaticTaxPayload(),
```

Comment:
I removed this because disabling automatic taxes during a swap is unlikely to happen. People can still do this by passing options explicitly if they want.